src/vs/workbench/common/editor.ts

```diff
@@ -440,6 +440,7 @@ export class EditorOptions implements IEditorOptions {
 		options.forceOpen = settings.forceOpen;
 		options.revealIfVisible = settings.revealIfVisible;
 		options.pinned = settings.pinned;
+		options.forcePreview = settings.forcePreview;
```

Comment:
I did not add it because TextDiffEditorOptions would ever be used in a preview content, correct?